picard/ui/metadatabox.py

```diff
@@ -585,14 +585,16 @@ def _update_tags(self, new_selection=True, drop_album_caches=False):
         top_tags = config.setting['metadatabox_top_tags']
         top_tags_set = set(top_tags)
 
+        settings = config.setting.as_dict()
+
         for file in files:
-            new_metadata = file.new_metadata
+            new_metadata = file.metadata
             orig_metadata = file.orig_metadata
             tags = set(list(new_metadata.keys()) + list(orig_metadata.keys()))
```

Comment:
Isn't this equivalent to:

```suggestion
            tags = set(new_metadata) | set(orig_metadata)
```

Reply:
Thanks for fixing this! Works as expected ;)